src/Documenter.jl

```diff
@@ -547,13 +694,22 @@ function deploydocs(;
         return
     end
 
+    local cibranch, pull_request, repo_slug, tag, event_type, finaldeploy
+
+    if haskey(ENV, "CI")
+        cibranch, pull_request, repo_slug, tag, event_type, finaldeploy = read_ci_env(true, uploader=uploader)
+    else
+        cibranch = devbranch
+        pull_request = false
+        repo_slug = repo
+        tag = isempty(tag) && read_ci_env()[4]
```

Comment:
This will set `tag` to `false` when `tag` is specified. You'd want to use the ternary `? :` here instead I think?

Do we want to rely on the CI variables here though, since this is the `haskey(ENV, "CI") === false` branch?

Reply:
True, I'll just not set tag here then.